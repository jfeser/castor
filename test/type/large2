    aorderedidx(select([o_orderdate], dedup(select([o_orderdate], orders))) as s7,
                      atuple([alist(select([l_extendedprice, l_discount],
                                      join(((o_orderdate = s7.o_orderdate) &&
                                           (((l_returnflag = "R")) &&
                                           (l_orderkey = o_orderkey))),
                                        lineitem,
                                        orders)) as s2,
                                atuple([ascalar(s2.l_extendedprice),
                                        ascalar(s2.l_discount)],
                                  cross))],
                        cross),
      >= param0, < (param0 + month(3)))
