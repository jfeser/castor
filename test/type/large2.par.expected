(OrderedIdxT
 ((DateT ((range (Interval 8036 10437))))
  (TupleT
   (((ListT
      ((TupleT
        (((FixedT ((value ((range Top) (scale 1)))))
          (FixedT ((value ((range Top) (scale 1))))))
         ((kind Cross))))
       ((count (Interval 1 3))))))
    ((kind Cross))))
  ((key_count (Interval 822 822)))))